data/data-pipeline/data_pipeline/comparison_tool/output/nyc_dacs/nyc_dacs__tract_comparison__2022-03-15.ipynb

```diff
@@ -0,0 +1,1779 @@
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th>Population</th>\n",
+       "      <th>Count of tracts</th>\n",
+       "      <th>Share of tracts</th>\n",
+       "      <th>Share of population</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>nyc_dac</th>\n",
+       "      <th>Definition M (communities)</th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th rowspan=\"2\" valign=\"top\">False</th>\n",
+       "      <th>False</th>\n",
+       "      <td>230,211,727.000</td>\n",
+       "      <td>50041</td>\n",
+       "      <td>0.675</td>\n",
+       "      <td>0.706</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>True</th>\n",
+       "      <td>89,094,870.000</td>\n",
+       "      <td>22398</td>\n",
+       "      <td>0.302</td>\n",
+       "      <td>0.273</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th rowspan=\"2\" valign=\"top\">True</th>\n",
+       "      <th>False</th>\n",
+       "      <td>2,579,235.000</td>\n",
+       "      <td>642</td>\n",
+       "      <td>0.009</td>\n",
+       "      <td>0.008</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>True</th>\n",
+       "      <td>4,404,139.000</td>\n",
+       "      <td>1079</td>\n",
+       "      <td>0.015</td>\n",
+       "      <td>0.013</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                        Population  Count of tracts  \\\n",
+       "nyc_dac Definition M (communities)                                    \n",
+       "False   False                      230,211,727.000            50041   \n",
+       "        True                        89,094,870.000            22398   \n",
+       "True    False                        2,579,235.000              642   \n",
+       "        True                         4,404,139.000             1079   \n",
+       "\n",
+       "                                    Share of tracts  Share of population  \n",
+       "nyc_dac Definition M (communities)                                        \n",
+       "False   False                                 0.675                0.706  \n",
+       "        True                                  0.302                0.273  \n",
+       "True    False                                 0.009                0.008  \n",
+       "        True                                  0.015                0.013  "
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# papermill_description=Summary_stats\n",
```

Comment:
I'm still really struggling with not using only the both-non-null geographies for the comparisons. The table above feels a little meaningless when it's like, we have 22,398 tracts in CEJST that aren't in NY's DACs and it's almost entirely because of the national scope of one but not the other.

Reply:
sorry - this was not using the updated comparison tool!